wcfsetup/install/lang/de.xml

```diff
@@ -3308,11 +3308,12 @@ freigeschaltet. {if LANGUAGE_USE_INFORMAL_VARIANT}Du kannst{else}Sie können{/if
 		<item name="wcf.attachment.attachments"><![CDATA[Dateianhänge]]></item>
 		<item name="wcf.attachment.delete.sure"><![CDATA[{if LANGUAGE_USE_INFORMAL_VARIANT}Willst du{else}Wollen Sie{/if} diesen Dateianhang wirklich löschen?]]></item>
 		<item name="wcf.attachment.upload.error.invalidExtension"><![CDATA[Die Datei hat eine ungültige Dateiendung.]]></item>
-		<item name="wcf.attachment.upload.error.tooLarge"><![CDATA[Die Datei ist zu groß.]]></item>
+		<item name="wcf.attachment.upload.error.tooLarge"><![CDATA[Die Dateigröße überschreitet die maximale Dateigröße.]]></item>
 		<item name="wcf.attachment.upload.error.reachedLimit"><![CDATA[{if LANGUAGE_USE_INFORMAL_VARIANT}Du hast{else}Sie haben{/if} die maximale Anzahl an Dateianhängen erreicht.]]></item>
 		<item name="wcf.attachment.upload.error.reachedRemainingLimit"><![CDATA[{if LANGUAGE_USE_INFORMAL_VARIANT}Du hast{else}Sie haben{/if} zu viele Dateianhänge ausgewählt, maximal verbleibend: #remaining#.]]></item>
 		<item name="wcf.attachment.upload.error.uploadFailed"><![CDATA[Beim Hochladen der Datei ist ein unbekannter Fehler aufgetreten.]]></item>
-		<item name="wcf.attachment.upload.error.uploadPhpLimit"><![CDATA[Die Datei ist größer als das PHP-Limit „upload_max_filesize“ und/oder „post_max_size“.]]></item>
+		<item name="wcf.attachment.upload.error.http413"><![CDATA[Die Dateigröße überschreitet das Limit der Anfragegröße des Webservers.]]></item>
```

Comment:
```suggestion
		<item name="wcf.attachment.upload.error.http413"><![CDATA[Die Dateigröße überschreitet das Limit der Anfragegröße des Webservers (HTTP-Statuscode 413).]]></item>
```


I think we should be a bit more specific here. After all this message is most likely the cause of a misconfiguration by the administrator, therefore it can be a bit more technical. The error message for PHP’s upload limit has proven to be extremely helpful, cutting down support requests for this significantly.